### ISSUE TITLE
Provide better error reporting for incorrect input

### ISSUE THREAD

**Issue by jeggy** [aPureBase/KGraphQL#93]
Currently this setup throws this exception: `java.lang.UnknownError: Something went wrong while searching for the constructor parameter type`
```kotlin
data class MyInput(val value1: String)

val schema = KGraphQL.schema {
    query("main") {
        resolver { input: MyInput -> "${input.value1}!!!"}
    }
}

schema.executeBlocking("""
    {
        main(input: { valu1: "Hello" })
    }
""".trimIndent()).let(::println)
```

It should throw a better error than just a `UnknownError` with no helpful message.

**Comment by Allali84** [comment]
What about `GraphQLError: Constructor Parameter 'valu1' can not be found in 'MyInput'` ?

**Comment by jeggy** [comment]
Sounds good 😃

**Comment by jeggy** [comment]
Has been solved and released in version `0.15.2`
